packages/common/src/ui/layout/tables/columns/NotePopoverColumn/NotePopoverColumn.tsx

```diff
@@ -23,19 +37,23 @@ export const getNotePopoverColumn = <T extends DomainObject>(
   align: ColumnAlign.Center,
   width: 60,
   accessor: ({ rowData }) => {
-    if (isObjectWithANote(rowData)) {
-      return rowData.note;
+    if (hasRequiredFields(rowData)) {
+      return rowData;
     } else {
       if (!isProduction()) {
         // TODO: Bugsnag during prod
         throw new Error(`
-        The default accessor for the note popover column has been called with a row
-        that does not have a note field.
-        Have you forgotten to provide a custom accessor to return a value? i.e.
-        { ...getNotePopoverColumn(), accessor: ({rowData}) => rowData.comment }
+        The default accessor for the note popover column has been called with row data
+        that does not have the fields 'header' and 'body'.
+
+        This column requires the fields 'header' and 'body' to be present in the row data to render
+        correctly.
+
+        Have you forgotten to provide a custom accessor to return the right value? i.e.
+        [ getNotePopoverColumn(), { accessor: ({rowData}) => ({header: rowData.batch, body: rowData.note}) }]
         `);
       } else {
-        return '';
+        return { header: '', body: '' };
```

Comment:
changed this to require the row data to have header and body fields rather than a `note` field.. easy enough to do a quick map in a custom accessor 🤷